articles/TOC.yml

```diff
@@ -1,68 +1,74 @@
 - name: Microsoft Quantum Documentation
   href: index.yml
-- name: Get started with quantum computing
+- name: Overview
   items:
-  - name: What is quantum computing?
-    href: overview/what-is-quantum-computing.md
-  - name: What can a quantum computer do?
-    href: overview/quantum-computers.md
-  - name: Why learn quantum computing?
-    href: overview/why-learn-quantum-computing.md
-  - name: What are Q# and the QDK?
-    href: overview/what-is-qsharp.md
-  - name: How can I learn more?
-    href: overview/how-to-learn-quantum-computing.md
-- name: Get the Quantum Development Kit
-  items:
-  - name: Get started with the QDK
+  - name: Get started
     href: welcome.md
+  - name: Five questions about quantum computing
+    items:
+    - name: What is quantum computing?
+      href: overview/what-is-quantum-computing.md
+    - name: What can a quantum computer do?
+      href: overview/quantum-computers.md
+    - name: Why learn quantum computing?
+      href: overview/why-learn-quantum-computing.md
+    - name: What is Q#?
+      href: overview/what-is-qsharp.md
+    - name: How to learn quantum computing?
+      href: overview/how-to-learn-quantum-computing.md
+  - name: Learn with the Quantum Katas
+    href: intro-to-katas.md
+- name: Quickstarts
+  items:
+  - name: Quantum random number generator
+    href: quickstarts/qrng.md
+  - name: Quantum basics with Q#
+    href: quickstart.md
+  - name: Implement Grover's search algorithm
+    href: quickstarts/search.md
+- name: How-to guides
+  items:
   - name: Install the QDK
     items:
     - name: Installation overview
       href: install-guide/index.md
+    - name: Q# standalone executable
```

Comment:
I think it's a little confusing to say that a user installs "Q# standalone executable," in that standalone executables are the compiler feature that allows users to work directly in Q# from the command line as opposed to using host programs in Python or C#, or to using Q# standalone from Jupyter.

Reply:
Probably should change the title "Install the QDK" to "Use the QDK" so that the standalone executable can fit in that part of the TOC

Reply:
I agree with Chris, it should just be something like "Q# command line applications".